src/matcha_ml/state/matcha_state.py

```diff
@@ -261,16 +306,26 @@ def is_local_state_stale(self) -> bool:
         local_config_file = os.path.join(os.getcwd(), "matcha.config.json")
 
         # the resource group name from the state object
-        matcha_state_resource_group = (
-            self.get_component("cloud").find_property("resource-group-name").value
-        )
 
-        if self.state_exists() and os.path.exists(local_config_file):
+        # matcha_state_resource_group: Optional[str] = (
+        #     self.get_component("cloud").find_property("resource-group-name").value
+        # )
+
```

Comment:
Required?

Reply:
No; my terrible habit of leaving commented code in. Thanks! Removed.